src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public interface Figure {

```

Comment:
You have a few options, you can define all these methods inside abstract class, because Abstract class defines general properties and functionality of child classes. Or you can create Interfaces, as to the Interface Segregation principal, because not every class who will implement getColor(), will have something for `area`. Like if you implement a car, Car doesn't need getArea(), but needs getColor()

Reply:
not fixed

Reply:
but i fixed that